vt: set toml file mod

micsthepick opened the pull request:
Set 0600 as the mode for the config file.

Commit:
vt: set toml file mod

micsthepick commented:
@plusvic I can certainly agree to a cla, but this repo has neither a security.md nor a contributing.md?

plusvic commented on vt/main.go:
```diff
@@ -41,13 +42,27 @@ func initConfig() {
+        cfg = filepath.Join(home, ".vt.toml")
+    }
+
+    if _, err := os.Stat(cfg); os.IsNotExist(err) {
```
I don't think this should be done here. I would put this in the `init` command, where the `.vt.toml` file is initially created.

https://github.com/VirusTotal/vt-cli/blob/eb5dbbc0f14cb654a76c2d24625a17c08b4949d6/cmd/init.go#L92

micsthepick replied:
would we still know where the config path is from though? the current solution grabs the correct path from the lib (maybe this is an upstream issue anyways?)

micsthepick replied:
I say that because it seems as if the path may be configured through command line args?

plusvic replied:
The path for the config file is always the user's home directory. My concern with the proposed change is that the file permissions are set on every execution of the tool, and the file is created if it doesn't exist. This shouldn't be tool's behavior in my opinion. The config file should be created only when the `init` command is executed, and I agree on setting 0600 permissions when the file is initially created, as it contains the user's API key and it shouldn't be visible to other others.

However, if the user later decides to change the initial permissions for whatever reason, the tool shouldn't be enforcing 0600 permission every time it runs.

plusvic committed:
refactor: don't change the config rule permissions on every execution of the tool.

Set 0600 permissions when the file is originally created. From that moment, if the user decides to change the initial permissions, the tool should not be messing up with manually assigned permissions. Also the tool should not create a `.vt.toml` file if it doesn't exist, except when the `init` command is run.

plusvic committed:
style: remove empty line.
